ui/component/app/view.jsx

```diff
@@ -24,7 +24,6 @@ import LANGUAGES from 'constants/languages';
 import { BeforeUnload, Unload } from 'util/beforeUnload';
```

Comment:
> _"Clear cache and hard reload".  Click the 3 publish-type buttons in succession._

- [ ] Should not transition to blank page while chunk is being loaded
    - I think this is more of a React/lazy-load/top-bar/router problem than a css one.  Can postpone this defect if not easily solvable.
    - Filling in Suspense's `fallback` with something cool would help a little, but not a solution.